Meetings/2022-08-16.md

```diff
@@ -0,0 +1,66 @@
+# PlanetaryPy TC Meeting August 16th @ 12:30 PM PST / 1:30 PM MST / 2:30 PM CST / 3:30 PM EST / 20:30 UTC
+
+* Zoom Link: https://cuboulder.zoom.us/j/2789149686
+
+## Participants
+
+* Michael Aye (MA)
+* Ross Beyer (RB)
+* Andrew Annex (AA)
+
+## pending AIs
+* setup poll for new meeting time
+
+## Agenda
+
+### craterstats
+
+* Michael met with Greg about craterstats
+* Greg doesn't mind a helping hand for the installation bugs
+* To avoid complexity with import_lib we should build it for Python 3.8, when everything was in the standardlib
+* I explained to Greg how the requirements.txt file needs to balance ease of life for developer against users
+  * Lots of patch-level version pinnings make it hard to install for the user, unless it only would ever be used in its own virtual environment
+* I also recommendend against pointing the user to any specific environment, as a novice user could interpret it as that kind of environment to be mandatory, which it rarely is.
+* After we made the package installable, MA will initiate/move forward the review process.
+* Lots of ideas on how to imprive the project, by making better use of the standard library, but we leave that for later.
+ 
+### Python version
+We agree to base all things (wherever easily possible and/or under our control) on Python 3.8 for convenience, as a lot of things run 3.9 or 3.10 already, so 3.8 is a convenient but already conservative-ish standard.
+
+
+* New logo updates from Benoit (if he's present)
+  * not present
+ 
+## AOB
+
+### TOML / package build process
+* AA mentions scikit-build as an interesting project that wants to [streamline] the build process for Python packages.
+  * This comes on top of the existing push to setup.cfg and pyproject.toml, and other changes at PyPA
+    * Also, recently, setup.py became obsolete for simple Python-only projects
+      * Still supported, but not required anymore 
+* AA: TOML becomes stdlib in 3.11, and is iffy to use beforehand as TOML libs are not "there" at install time for the older versions
+
+### PDS4
+* Question came up if the specifics of PDS4 were still a "moving target", and RB thinks they are, specifically the data dictionaries that are required to constrain the "volatility" of XML.
+* These data dicts aren't the same between missions.
```

Comment:
That last sentence in line 45 is not what I meant.  In the old PDS3 days, there was not tight control on the key-names in PDS3 labels.  The PDS4 Data Dictionaries provide that control, and Missions can define their own Data Dictionaries (for those items not already covered by the generic higher-level PDSData Dictionaries).  The trouble is that the PDS Data Dictionaries are undergoing a lot of change and flux, so current implementors are having a hard time tracking upstream, so to speak.